modules/core/test/v2/unit/internal/tssUtils.ts

```diff
@@ -485,6 +485,17 @@ describe('TSS Utils:', async function () {
     });
   });
 
+  describe('delete SignatureShare:', async function() {
```

Comment:
Do we have failure cases?

Reply:
![image](https://user-images.githubusercontent.com/64786258/156436466-c9900a44-84da-4357-ab47-e09cfe78b17f.png)

It calls this endpoint, I think there is not much to test from core